### DISCUSSION TITLE
Can not launch it with Remix >= v2.6

### DISCUSSION THREAD

**Discussion by Mat-moran** [Code-Forge-Net/Remix-Dev-Tools#100] [Q&A] [answered]
Hi I am just following your tutorial in https://www.youtube.com/watch?v=gG8BVFyVBmY but when I launch the app, nothing appears in the remix tab.

![image](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/4810013/5fc29494-8bf4-4e9a-90ae-4096b1266b02)

![image](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/4810013/a56f5624-c349-4f29-ace7-f9e573355393)

![image](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/4810013/a9bf3be4-69fd-4e41-87b1-4a4159ef0cf5)

This is an error message in the console log, I do not know if it is related
![image](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/4810013/c4df91d8-ed6f-48e0-a05c-4dec5f50028b)

Do I need to configure something else?

Thanks and congrats for this tool! 😄

**Comment by Mat-moran** [comment, marked as answer]
Hi in this new version of remix it is necessary to put the `remixDevTools()` plugin before the `remix()` plugin in `vite.config.ts`

**Comment by AlemTuzlak** [comment]
@Mat-moran I plan to add an error if this scenario happens, this wasn't needed before but is now, will probably throw an error if the order is not correct
